README.md

```diff
@@ -1,53 +1,290 @@
+# What is Nextcloud?
+
 [![Build Status update.sh](https://doi-janky.infosiftr.net/job/update.sh/job/nextcloud/badge/icon)](https://doi-janky.infosiftr.net/job/update.sh/job/nextcloud)
 [![Build Status Travis](https://travis-ci.org/nextcloud/docker.svg?branch=master)](https://travis-ci.org/nextcloud/docker)
 
-What is Nextcloud?
-
 A safe home for all your data. Access & share your files, calendars, contacts, mail & more from any device, on your terms.
 
 ![logo](https://github.com/nextcloud/docker/raw/master/logo.png)
 
 # How to use this image
-This image is designed to be used in a micro-service environment. It consists of the Nextcloud installation in an [php-fpm](https://hub.docker.com/_/php/) container. To use this image it must be combined with any webserver that can proxy the http requests to the FastCGI-port of the container.
+This image is designed to be used in a micro-service environment. There are two versions of the image you can choose from.
+
+The `apache` tag contains a full nextcloud installation including an apache web server. It is designed to be easy to use and get's you running pretty fast. This is also the default for the `latest` tag and version tags that are not further specified.
 
-## Start Nextcloud
+The second option is a `fpm` container. It is based on the [php-fpm](https://hub.docker.com/_/php/) image and runs a fastCGI-Process that serves your nextcloud page. To use this image it must be combined with any webserver that can proxy the http requests to the FastCGI-port of the container.
 
-Starting Nextcloud php-fpm instance listening on port 9000 is as easy as the following:
+## Using the Apache image
+The apache image contains a webserver and exposes port 80. However by default it is not configured to use ssl encryption. To start the container type:
 
 ```console
 $ docker run -d nextcloud
 ```
 
-Now you can get access to fpm running on port 9000 inside the container. If you want to access it from the internet, we recommend using a reverse proxy in front. You can install it directly on your machine or use an additional container (You can find more information on that on the docker-compose section). Once you have a reverse proxy, you can access Nextcloud at http://localhost/ and go through the wizard. 
+Now you can access Nextcloud at http://localhost/ from your host system. To make your nextcloud installation available from the internet you must map the port of the container to your host:
 
+```console
+$ docker run -p 80:80 -d nextcloud
```

Comment:
Each part contains a line about how that setup does **not** contain ssl encryption, I can make it bold if you like 😁. Since nextcloud itself complains about unsecure connection when you connect via http, I think it should be fine. 

I'll have a look at removing port mappings from the example and move it to a new part as you proposed.